sympy/core/mod.py

```diff
@@ -209,16 +214,16 @@ def doit(p, q):
         # check again to see if p and q can now be handled as numbers
         rv = doit(p, q)
         if rv is not None:
-            return rv*G
+            return rv * G
 
         # put 1.0 from G on inside
         if G.is_Float and G == 1:
             p *= G
             return cls(p, q, evaluate=False)
         elif G.is_Mul and G.args[0].is_Float and G.args[0] == 1:
-            p = G.args[0]*p
+            p = G.args[0] * p
```

Comment:
It looks like your editor has automatically changed some other things in this file.